cloudify_cli/commands/profiles.py

```diff
@@ -187,6 +191,21 @@ def use(manager_ip,
             logger=logger,
             **kwargs)
 
+    active_profile = _get_profile(env.get_active_profile())
+    active_profile_context = env.get_profile_context(env.get_active_profile())
+    if active_profile_context.manager_username is None:     # docl 1st run
+        return
+    client = get_rest_client(
```

Comment:
Using the `get_rest_client` would use the current profile, in case no client_profilem is given to the function. Moreover `get_rest_client` resolves the `rest_port`, `rest_protocol`, `rest_certificate`, and `tenant_name` the same as you've done here.

Reply:
And a better practice would be using the `get` function in these cases, i.e. `active_profile[x]` => `active_profile.get(x)`. 
In that way we avoid key errors, in case there are ones.